pyvcloud/vcd/org.py

```diff
@@ -496,6 +472,39 @@ def create_user(self,
             EntityType.USER.value,
             user)
 
+    def get_user(self, user_name):
+        """
+        Retrieve user record from current Organization
+        :param user_name: user name of the record to be retrieved
+        :return: User record
+        """ # NOQA
+        if self.resource is None:
+            self.resource = self.client.get_resource(self.href)
+        resource_type = 'adminUser' if self.is_admin else 'user'
+        org_filter = 'org==%s' % self.resource.get('href') \
+            if self.is_admin else None
+        query = self.client.get_typed_query(
+            resource_type,
+            query_result_format=QueryResultFormat.REFERENCES,
+            equality_filter=('name', user_name),
+            qfilter=org_filter
+            )
+        records = list(query.execute())
+        if len(records) == 0:
+            raise Exception('user not found')
+        elif len(records) > 1:
+            raise Exception('multiple users found')
+        return self.client.get_resource(records[0].get('href'))
+
+    def delete_user(self, user_name):
+        """
+        Delete user record from current organization
+        :param user_name: (str) name of the user that (org/sys)admins wants to delete
+        :return: result of calling DELETE on the user resource
+        """ # NOQA
+        user = self.get_user(user_name)
+        return self.client.delete_resource(user.get('href'))
```

Comment:
Where are we deleting user associated entities like vApps, vms, catalogs as described in the vcd-cli command description? Does it force delete automatically?

Reply:
vCD does it for us. We don't have to separately delete the entities.